[MI-3238] Update documentation (#7)
Brightscout/mattermost-load-test-scripts: README.md

Mattermost load-test-scripts provides a set of scripts written in [Go](https://golang.org/) to help profiling [Mattermost](https://github.com/mattermost/mattermost-server) under heavy load, simulating real-world usage of a server installation at scale.

## Setup

Make sure you have the following components installed:  

- Go - v1.18 - [Getting Started](https://golang.org/doc/install)

>     > **Note:** If you have installed Go to a custom location, make sure the `$GOROOT` variable is set properly. Refer [Installing to a custom location](https://golang.org/doc/install#install).

Reviewer: ```suggestion
    > **Note:** If you have installed 'Go' to a custom location, make sure the `$GOROOT` variable is set properly. Refer [Installing to a custom location](https://golang.org/doc/install#install).
```

    - Configure the `config.json` file created according to the load to be tested.
    - Go to config [docs](docs/config.md) to check details on the config settings.

- Run the command `make build` to create a new binary file for the load test scripts.

- Run the command `make create_users` to create new users with the details present in the config file.

- Connect the system admin account with an MS Teams account to link the new channels with the MS Teams channels.

- Run the command `make create_channels` to create the new channels and add the new users to them. Running this command also links the Mattermost channels with the MS Teams channels present in the config file.

- Run the command `make create_dm_and_gm` to create the DMs and GMs between the new users.

- Login and connect all the new users with their respective MS Teams accounts to enable the relaying of messages from Mattermost to MS Teams on behalf of these users.

- Run the command `make create_posts` to create the random posts in the Mattermost channels, DMs, and GMs.

> - Run the command `make clear_store` to clear all the stored data present in the temporary database to start load testing with new details.

Reviewer 2: ```suggestion
- Run the command `make clear_store` to clear all the data present in the temporary file called `temp_store.json` to start load testing with new details.
```

- Login and connect all the new users with their respective MS Teams accounts to enable the relaying of messages from Mattermost to MS Teams on behalf of these users.

- Run command `make create_posts` to create the random posts in the Mattermost channels, DMs, and GMs.

> - Run command `make clear_store` to clear all the stored data present in the temporary database to start load testing with new details.

Reviewer 3: ```suggestion
- Run the command `make clear_store` to clear all the stored data present in the temporary database to start load testing with new details.
```
